src/main/java/edu/wpi/first/pathui/QuickSpline.java

```diff
@@ -0,0 +1,80 @@
+package edu.wpi.first.pathui;
+
+import java.util.Map;
+
+import javafx.geometry.Point2D;
+import javafx.scene.Group;
+import javafx.scene.input.Dragboard;
+import javafx.scene.input.TransferMode;
+import javafx.scene.shape.CubicCurve;
+import javafx.util.Pair;
+
+public class QuickSpline implements ISpline {
+  private final Waypoint start;
+  private Waypoint end;
+  private final CubicCurve curve;
+
+  @Override
+  public void setEnd(Waypoint end) {
+    this.end = end;
+    update();
+  }
+
+  @Override
+  public void removeFromGroup(Group splineGroup) {
+    splineGroup.getChildren().remove(curve);
+  }
+
+  /**
+   * Creates a Spline that is "Quick", i.e. a JavaFX CubicCurve.
```

Comment:
The definition of a "quick" spline should go in the class-level docs, not the constructor. Also needs clarification on what a "slow" spline would be